src/app.js

```diff
@@ -1,8 +1,13 @@
+require('dotenv').config({path:'./env'})
+require("express-async-errors");
 const express = require('express');
 const app = express();
 const cors = require('cors')
 const favicon = require('express-favicon');
 const logger = require('morgan');
+const express = require("express");
```

Comment:
notFoundMiddleware and errorHandlerMiddleware need to be imported here before being used

Reply:
Also made sure to import these